influence_benchmark/config/accelerate_config.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
```

Comment:
should probably add a multi gpu version of this as well

Reply:
Yeah, I guess do you have any that we can test? I don't think multi-gpu was working for me on RNN even without SLURM? I'll try again